packages/ra-ui-materialui/src/input/TranslatableInputsTabContent.tsx

```diff
@@ -23,7 +18,11 @@ export const TranslatableInputsTabContent = (
     props: TranslatableInputsTabContentProps
 ): ReactElement => {
     const { children, groupKey = '', locale, ...other } = props;
-    const { selectedLocale, getLabel, getSource } = useTranslatableContext();
+    const { selectedLocale, getSource } = useTranslatableContext();
```

Comment:
Unless I'm mistaken, we should mark `getLabel` as deprecated in `TranslatableContext`

Reply:
I haven't changed the `TranslatableFields` yet which still uses it. Should I update it in this PR as well?

Reply:
Good question. I'd advocate for a second PR since this one is already big enough. Can you add a card in our internal tracker (including the need to mark getLabel as deprecated)?

Reply:
It's not deprecated though